src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -377,11 +376,28 @@ public Path getJarPath(String jarName) {
   }
 
   /**
-   * Checks whether the Cloud SDK Path with is valid.
+   * Checks whether the Cloud SDK path and version are valid.
    *
-   * @throws CloudSdkNotFoundException when the Cloud SDK is not installed where expected
+   * @throws CloudSdkNotFoundException when an up-to-date Cloud SDK is not installed where expected
    */
-  public void validateCloudSdk() throws CloudSdkNotFoundException {
+  public void validateCloudSdk() {
+    validateCloudSdkLocation();
```

Comment:
In IntelliJ at least, if I 'surround with try/catch' on a block, it'll automatically put the exceptions listed in the method declaration into the catch.  It does not do that for exceptions listed in @throws.